dev/app/custom-widgets/jquery/currentRefinedValues.js

```diff
@@ -46,17 +46,8 @@ const renderFn = ({
     // ---------------------------
     const list = refinements
       .map(value => {
-        if (value.hasOwnProperty('operator') && typeof value.operator === 'string') {
-          let displayedOperator = value.operator;
-          if (value.operator === '>=') displayedOperator = '&ge;';
-          if (value.operator === '<=') displayedOperator = '&le;';
-          value.name = `${displayedOperator} ${value.name}`;
-        }
-
-        return value;
-      })
-      .map(value => {
-        const {name, count} = value;
+        const {labelWithOperator, count} = value;
+        const name = labelWithOperator || value.name;
```

Comment:
Doesn't have always an operator, renamed `labelWithOperator` to `computedLabel` which defaults to `value.name`

see 0a6dbbf5